docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2175,6 +2176,30 @@ to_unixtime(expression[, ..., format_n])
 - [unnest](#unnest)
```

Comment:
Please add a bulletin for `list_array_any`.

Reply:
Done